apps/vr-tests/src/stories/ComboBox.stories.tsx

```diff
@@ -55,38 +42,22 @@ storiesOf('ComboBox', module)
         .click('.ms-Button-flexContainer')
         .hover('.ms-Button-flexContainer')
         .snapshot('click', { cropTo: '.ms-Layer' }) // Dropdown extends beyond testWrapper
-        .end()
-      }
+        .end()}
     >
       {story()}
     </Screener>
   ))
-  .addStory('Root', () => (
-    <ComboBox
-      defaultSelectedKey='A'
-      label='Default with dividers'
-      ariaLabel='Basic ComboBox example'
-      autoComplete='on'
-      options={testOptions}
-    />
-  ), { rtl: true })
+  .addStory('Root', () => <ComboBox defaultSelectedKey="A" label="Default with dividers" autoComplete="on" options={testOptions} />, {
+    rtl: true
+  })
   .addStory('Styled', () => (
     <ComboBox
-      defaultSelectedKey='A'
-      label='Styled with dividers'
-      ariaLabel='Basic ComboBox example'
-      autoComplete='on'
+      defaultSelectedKey="A"
+      label="Styled with dividers"
```

Comment:
Possibly because it was wrong (didn't match the label). I actually found the usage of ariaLabel in both examples and tests for this component odd...I'm pretty sure that you don't need an explicit aria-label as long as the component itself has an adequately descriptive label and the component id/label for are set up properly (which our components do automatically).